pkgs/servers/nosql/apache-jena/fuseki-test.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, runCommandNoCC, apache-jena-fuseki, curl }:
```

Comment:
```
{ lib, runCommand, apache-jena-fuseki, curl }:
```
This is an alias and breaks eval. Please fix asap!

Reply:
> Does this need fixing in staging-next or master?

master

Reply:
https://github.com/NixOS/nixpkgs/pull/247699